news/2022-10-03-results-triangles.md

```diff
@@ -0,0 +1,153 @@
+
+![Overall winner](/wiki/shared/news/2022-10-03-results-triangles/overall.jpg)
+
+[![submission preview](/wiki/shared/news/2022-10-03-results-triangles/winnerpreview.png)](https://osu.ppy.sh/beatmapsets/1841885)
+
+Honestly, words don't do this justice. [Play the map yourself](https://osu.ppy.sh/beatmapsets/1841885) and soak in the storyboard to fully understand why it's the winner.
+
+[Play it.](https://osu.ppy.sh/beatmapsets/1841885)
+
+[You won't regret it.](https://osu.ppy.sh/beatmapsets/1841885)
+
+[We promise.](https://osu.ppy.sh/beatmapsets/1841885)
+
+Each member of this team has earned the **Overall winner** profile badge, and its participants have earned **Elite Mapper** and **Elite Storyboarder** user titles accordingly!
+
+![Overall winner profile badge](/wiki/shared/news/2022-05-26-triangles/overallwinner@2x.png)
+
+We can't forget the grand prize of this contest: **the winning submission is bundled with all iterations of osu!(lazer) going forward.**
+
+Once ranked, this beatmap will be one of the first things an osu! player sees in the new client! Given its sheer quality in every aspect, there's no better map for the role.
+
+Major props to everyone involved in this future classic!
+
+## Full results
+
+If you want all the juicy judging details, you're in the right place.
+
+Here's the breakdown of scores:
+
+| Rank | Team submission | Mapper panel | Storyboarder panel | Averaged total |
+| :-- | :-- | :-- | :-- | :-- |
+| \#1 | [Crazy Thursday](https://osu.ppy.sh/beatmapsets/1841885) | 143 | 127 | **254.11** |
+| \#2 | [Crush the Candy Boys](https://osu.ppy.sh/beatmapsets/1841845) | 142 | 105 | **231.22** |
+| \#3 | [三角形.tar.gz](https://osu.ppy.sh/beatmapsets/1842041) | 84 | 149 | **223.67** |
+| \#4 | [absolutno pofig](https://osu.ppy.sh/beatmapsets/1841846) | 81 | 101 | **173** |
+| \#5 | [Δ = b² – 4ac](https://osu.ppy.sh/beatmapsets/1842004) | 78 | 84 | **153.33** |
+| \#6 | [?](https://osu.ppy.sh/beatmapsets/1841847) | 51 | 106 | **151.33** |
+| \#7 | [TV Sizers](https://osu.ppy.sh/beatmapsets/1841923) | 115 | 48 | **150.22** |
+| \#8 | [ULTRA ICE TEA](https://osu.ppy.sh/beatmapsets/1841926) | 68 | 84 | **144.44** |
+| \#9 | [Team Funny](https://osu.ppy.sh/beatmapsets/1842149) | 88 | 26 | **104.2222222** |
+| \#10 | [GokuLook Princesa](https://osu.ppy.sh/beatmapsets/1841839) | 69 | 42 | **103.33** |
+| \#11 | [Penta-force](https://osu.ppy.sh/beatmapsets/1841996) | 36 | 35 | **67** |
+| \#12 | dreamers | 9 | 8 | **16** |
+
+Judging for this contest involved some experimentation. Instead of assigning raw numbers to submission like most contests, the judges were given two tasks:
```

Comment:
```suggestion
Judging for this contest involved some experimentation. Instead of assigning raw numbers to submissions like most contests, the judges were given two tasks:
```